components/UI/iconsComponents/icons/customCheckMark.tsx

```
@@ -0,0 +1,13 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+
+// custom Checkmark icon from new Figma design
+const CustomCheckmarkIcon = (props: SvgIconProps) => (
+  <SvgIcon {...props} >
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)